mix.exs

```diff
@@ -54,17 +54,15 @@ defmodule Scholar.MixProject do
       logo: "images/scholar_simplified.png",
       extra_section: "Guides",
       extras: [
+        # "notebooks/hierarchical_clustering.livemd",
         "notebooks/linear_regression.livemd",
         "notebooks/k_means.livemd",
         "notebooks/k_nearest_neighbors.livemd",
         "notebooks/cv_gradient_boosting_tree.livemd"
       ],
       groups_for_modules: [
         Models: [
-          Scholar.Cluster.AffinityPropagation,
-          Scholar.Cluster.DBSCAN,
-          Scholar.Cluster.GaussianMixture,
-          Scholar.Cluster.KMeans,
+          ~r/Scholar.Cluster/,
```

Comment:
This is a bit of code golf. Doing it this way has side effects, so I'm not sure if it's desired.

Reply:
I like this approach but we need to escape dots. Perhaps it is better done in a separate PR too:

```suggestion
          ~r/Scholar\.Cluster\./,
```

Reply:
> Perhaps it is better done in a separate PR too

👍 Yeah I think that's better. Will change.

Reply:
[2f2de8e](https://github.com/elixir-nx/scholar/pull/187/commits/2f2de8e36ae560fe37bba10e41609b21dfc79954)